src/main/java/com/google/sps/data/TextAnalyser.java

```diff
@@ -210,45 +208,86 @@ public AnalyzeSyntaxResponse analyseSyntaxText() throws IOException {
     }
   }
 
-  public Set<String> getAdjectives() throws IOException {
+  // Get the combination of verb-adverb
+  public Set<String> getSyntax() throws IOException {
     Set<String> adjectives = new LinkedHashSet<String>();
+    String verb = "";
+    boolean add = false;
 
     for (Token token : analyseSyntaxText().getTokensList()) {
       String partOfSpeech = token.getPartOfSpeech().getTag().toString();
-      if(partOfSpeech.equals("ADJ")) {
-        adjectives.add(token.getLemma().toLowerCase());
+      
+      if(partOfSpeech.equals("VERB") && !add) {
+        verb = token.getText().getContent().toLowerCase();
+        add = true;
```

Comment:
You could consider writing two for-loops instead of one. The first one to search for the verb and the second one for the adjective in case a verb had been found. This might be easier to read and the loop here is cheap.